insteon_mqtt/device/Base.py

```diff
@@ -60,7 +60,6 @@ def from_config(cls, values, protocol, modem, **kwargs):
         for config in values:
             # If it's a dict, it's got a nice name set.
             if isinstance(config, dict):
-                assert len(config) == 1
```

Comment:
```yaml
- 3a.29.84: 'lamp2'
  on_off_ramp_supported: True
```

While concise, I don't think this config design works.  There is no order to a dict in python and as such we can't be sure that the first key/val pair is the addr/name.  In which case, the line below this comment would fail.

So if we end up sticking with hardcoded configs (or if we think this functionality may be useful in the future) we need to fix this.

I used something like the following in scenes, the 3 examples of how a device may be defined in the yaml file.  My programmer side would prefer just to use the long version every time since there is no ambiguity, but I recognize that for user happiness we probably need all three and just have to deal with the extra code.
```yaml
- DE.AD.BE
- EF.01.02: Device2
- 03,04.05:
    name: Device3
    on_off_ramp_supported: True
```
if
1. the config is not a dict, it is just the addr.
2. if the config is a dict, it must be a dict of length 1
2a. In that case if the val is not a dict, it is assume to be the name
2b. if the val is a dict, it is expected to have a name key in the dict

Reply:
Resolving, as these config-related changes have been reverted in this PR.